src/utils/Utils.ts

```diff
@@ -1,28 +1,35 @@
-import { ResourceType, ResourceTypeAttribute, ResourceInstance } from '@/apis/rembrandt/rembrandt';
-import { ListElement } from '@/components/ListSection.vue';
+import { ResourceType, ResourceTypeAttribute } from '@/apis/rembrandt/rembrandt';
+import { LiOptions } from '@/components/Li.vue';
+
+export type clickHandler = (id: string) => void;
 
 export default class Utils {
   // region public static methods
-  public static resourceTypesToList(resourceTypes: ResourceType[]): ListElement[] {
+  public static resourceTypesToList(resourceTypes: ResourceType[], onClick?: clickHandler): LiOptions[] {
```

Comment:
something about the naming is odd, before it was singular (ListElement), now it is plural (LiOptions)

Reply:
Is resolved and renamed to ListEntry.